docs/website/content/v0.3/en/guides/cloud/aws.md

```diff
@@ -2,18 +2,225 @@
 title: 'AWS'
 ---
 
-First, create the AMI:
+## Creating a Cluster via the AWS CLI
+
+In this guide we will create an HA Kubernetes cluster with 3 worker nodes.
+We assume an existing VPC.
+
+### Create the Subnet
+
+```bash
+aws ec2 create-subnet \
+    --region ${REGION} \
+    --vpc-id ${VPC} \
+    --cidr-block 10.0.1.0/24
+```
+
+### Create the AMI
+
+#### Prepare the Import Prerequisites
+
+##### Create the S3 Bucket
+
+```bash
+aws s3api create-bucket \
+    --bucket talos-aws-tutorial-${ACCOUNT} \
+    --create-bucket-configuration LocationConstraint=${REGION} \
+    --acl private
+```
+
+##### Create the `vmimport` Role
+
+In order to create an AMI, ensure that the `vmimport` role exists as described in the [official AWS documentation](https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role).
```

Comment:
May want to add a note / sentence saying that this role should be associated with the bucket created above ( `talos-aws-tutorial-${ACCOUNT}` ) just to highlight/reiterate the purpose.

Reply:
Good point.